broadcast/index.md

```diff
@@ -38,8 +38,22 @@ weeks!
 # Upcoming shows
  <ul>
```

Comment:
Can you do this list using normal Markdown instead of HTML?

Reply:
Done.